SQL/0000-00-04-Help.sql

```diff
@@ -61,14 +60,20 @@ The resulting table:\n
+(42,-1,md5('examiner'),'Examiner','The Examiner tab allows the authorized user to add, view, or modify examiners and their certifications.\r\n To add an examiner, the name and site of the examiner must be specified, as well as whether or not the examiner is a radiologist. \r\n The Examiner Module has a Selection Filter function to allow users to search for a particular examiner and/or site. By clicking the button Show Data after selecting certain search options, the search results will appear, organized by the blue headers Examiner, Site, Radiologist, and Certification. \r\n By clicking on an examiner\'s name, the certification for that examiner can be added or modified. To edit certification for an examiner, choose an instrument under the  Instrument header, click the Certification Status drop-down, select the correct Certification Date and enter in any pertinent comments. Any modifications to an examiner\'s certification will appear in the Change Log.', '2016-02-23 00:00:00', NULL),
+(43,42,md5('edit_examiner'),'Edit Examiner','To edit certification for an examiner, choose an instrument under the  Instrument header, click the Certification Status drop-down, select the correct Certification Date and enter in any pertinent comments. Any modifications to an examiner\'s certification will appear in the Change Log.','2016-02-23 00:00:00', NULL),
+(44,-1,md5('training'),'Training','The Training Module allows users to view training content in three columns:
+Certifications to Complete, Completed Certifications, and Online Training. \r\n Certifications to Complete consists of available existing training modules that the user has not yet completed. Completed Certifications allow the user to browse the training content for any of the instruments that the user has already been certified for. Online Training allows the user to access training for certain instruments online.','2016-02-23 00:00:00', NULL),
```

Comment:
instrument_list is already there, but I noticed in demo and ibis don't have so I'm assuming it was added afterwards at some point